drivers/hmis/app/graphql/types/hmis_schema/enums/project_type.rb

```diff
@@ -11,19 +11,11 @@ class HmisSchema::Enums::ProjectType < Types::BaseEnum
     description 'HUD Project Types'
     graphql_name 'ProjectType'
 
-    value 'ES', 'Emergency Shelter', value: 1
-    value 'TH', 'Transitional Housing', value: 2
-    value 'PSH', 'Permanent Supportive Housing', value: 3
-    value 'SO', 'Street Outreach', value: 4
-    value 'SERVICES_ONLY', 'Services Only', value: 6
-    value 'OTHER', 'Other', value: 7
-    value 'SH', 'Safe Haven', value: 8
-    value 'OPH', 'Permanent Housing Only', value: 9
-    value 'PH', 'Permanent Housing', value: 10
-    value 'DAY_SHELTER', 'Day Shelter', value: 11
-    value 'PREVENTION', 'Homelessness Prevention', value: 12
-    value 'RRH', 'Rapid Re-Housing', value: 13
-    value 'CE', 'Coordinated Entry', value: 14
+    HudUtility.project_types.each do |id, description|
```

Comment:
Did you want HudUtility2024?

Reply:
not yet - doing that in  https://github.com/greenriver/hmis-warehouse/pull/3383